build_tools/mako/config/mobile-bert-s20.config

```diff
@@ -0,0 +1,45 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+benchmark_key: "4699630718681088"
+benchmark_name: "MobileBert (S20) (Time Unit: ms)"
+project_name: "iree"
+
+# Only owners can write to the benchmark
+owner_list: "hanchung@google.com"
+owner_list: "buildkite-agent@iree-oss.iam.gserviceaccount.com"
+
+# Define the name and type for x-axis of run charts
+input_value_info: {
+  value_key: "t"
+  label: "time"
+  type: NUMERIC
+}
+
+# Three metrics, define the names for y-axis values of both run and aggregate
+# charts.
+metric_info_list: {
+  value_key: "cpu"
+  label: "DYLib_AOT"
+}
+metric_info_list: {
+  value_key: "vmla"
+  label: "VMLA"
+}
+metric_info_list: {
+  value_key: "vlk"
```

Comment:
Why not just using `vulkan`?

Reply:
We have this discussion before, see https://github.com/google/iree/pull/3851#discussion_r528287625

They suggest to use one or two chars: https://github.com/google/mako/blob/master/spec/proto/mako.proto#L91
We end up with using ~3 characters so name it vlk.